_posts/2024-01-03-opensearch-performance-improvements.md

```diff
@@ -0,0 +1,1151 @@
+---
+layout: post
+title: An update on the OpenSearch Project’s continued performance progress through 2.11 
+authors:
+  - sisurab
+  - pallp
+  - dagney
+  - rishabhsi
+date: 2024-01-03
+categories:
+    - technical-posts
+    - community
+meta_keywords: 
```

Comment:
Please add the following meta: 

**meta_keywords:**  OpenSearch performance improvements, OpenSearch roadmap, high volume indexing, low latency search
**meta_description:** Learn more about the OpenSearch Project roadmap and how the project improved the performance of its core open source engine to drive down latency and improve efficiency.